great_expectations/datasource/data_connector/configured_asset_glue_catalog_data_connector.py

```diff
@@ -0,0 +1,140 @@
+        """
+        logger.warning(
+            "Warning: great_expectations.datasource.data_connector.ConfiguredAssetGlueCatalogDataConnector is "
+            "experimental. Methods, APIs, and core behavior may change in the future."
+        )
+        super().__init__(
+            name=name,
+            datasource_name=datasource_name,
+            execution_engine=execution_engine,
+            assets=assets,
+            batch_spec_passthrough=batch_spec_passthrough,
+        )
+
+        if boto3_options is None:
+            boto3_options = {}
+
+        try:
+            self._glue = boto3.client("glue", **boto3_options)
+        except (TypeError, AttributeError):
+            raise ImportError(
+                "Unable to load boto3 (it is required for ConfiguredAssetGlueCatalogDataConnector)."
+            )
+
+    @property
+    def execution_engine(self) -> SparkDFExecutionEngine:
+        return cast(SparkDFExecutionEngine, self._execution_engine)
+
+    def _update_data_asset_name_from_config(
+        self, data_asset_name: str, data_asset_config: Optional[dict]
+    ) -> str:
+        if data_asset_config is None:
+            data_asset_config = {}
+
+        data_asset_name_prefix: str = data_asset_config.get(
+            "data_asset_name_prefix", ""
+        )
+        data_asset_name_suffix: str = data_asset_config.get(
+            "data_asset_name_suffix", ""
+        )
+
+        data_asset_name = (
+            f"{data_asset_name_prefix}{data_asset_name}{data_asset_name_suffix}"
+        )
+        return data_asset_name
+
+    def _get_batch_identifiers_list_from_data_asset_config(
+        self,
+        data_asset_name: str,
+        data_asset_config: Optional[dict],
+    ) -> List[dict]:
+        return [{}]
+
+    def build_batch_spec(
+        self, batch_definition: BatchDefinition
+    ) -> GlueCatalogBatchSpec:
+        """
+        Build BatchSpec from batch_definition by calling DataConnector's build_batch_spec function.
+
+        Args:
+            batch_definition (BatchDefinition): to be used to build batch_spec
+
+        Returns:
+            BatchSpec built from batch_definition
+        """
+        batch_spec: BatchSpec = super().build_batch_spec(
+            batch_definition=batch_definition
+        )
+        return GlueCatalogBatchSpec(batch_spec)
+
+    def _get_table_name_from_batch_definition(
+        self, batch_definition: BatchDefinition
+    ) -> str:
+        """
+            Helper method called by _get_batch_identifiers_list_from_data_asset_config() to parse table_name from data_asset_name in cases
+            where database_name is included.
+
+            data_asset_name in those cases are [database_name].[table_name].
+
+        function will split data_asset_name on [database_name]. and return the resulting table_name.
+        """
+        data_asset_dict: dict = self.assets[batch_definition.data_asset_name]
```

Comment:
@lccasagrande Similarly to an earlier comment, could we not simply make this method in the parent class "stronger" and then not override it in the present class?  Thank you.

Reply:
I did a code refactoring and I am not using the **ConfiguredAssetSqlDataConnector** as parent class anymore. I found it better to inherit from the **DataConnector** class.